src/coreclr/vm/threadsuspend.cpp

```diff
@@ -2877,7 +2877,7 @@ BOOL Thread::RedirectThreadAtHandledJITCase(PFN_REDIRECTTARGET pTgt)
     // This should not normally fail.
     // The system silently ignores any feature specified in the FeatureMask
     // which is not enabled on the processor.
-    bRes &= SetXStateFeaturesMask(pCtx, XSTATE_MASK_AVX);
+    SetXStateFeaturesMask(pCtx, XSTATE_MASK_AVX);
```

Comment:
Did this actually return a failure?

Reply:
I think the actual implementation would filter the mask with available features and then apply, thus it would not fail for unavailable features. 
 Maybe that is different under the emulation.

Reply:
Yes, it always returned `FALSE` and `GetLastError` returned `ERROR_INVALID_PARAMETER`.